DataLakeModels/Models/YouTube/Studio/Group.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+
+namespace DataLakeModels.Models.YouTube.Studio {
+
+    public class Group : IEquatable<Group> {
+        public Group() {}
+
+        public string GroupId { get; set; }
+        public string ChannelId { get; set; }
+        public string Title { get; set; }
+        public DateTime RegistrationDate { get; set; }
+        public DateTime? UpdateDate { get; set; }
```

Comment:
Does it make sense to have a null UpdateDate? Couldn't we just set to the RegistrationDate?